db/schema.rb

```diff
@@ -1360,14 +1360,13 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
-    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Comment:
was this supposed to be committed with these changes?

Reply:
Nooo, thanks, I fixed the schema :)